railties/lib/rails/generators/rails/sessions/sessions_generator.rb

```diff
@@ -16,7 +16,7 @@ def create_session_files
 
       def configure_application
         gsub_file "app/controllers/application_controller.rb", /(class ApplicationController < ActionController::Base)/, "\\1\n  include Authentication"
-        route "resource :session"
+        route "resource :session, only: %i[new create destroy]"
```

Comment:
Not interested in this change. No need to narrow this down. It pollutes the routes.rb file and the benefits are dubious.

Reply:
Since Rails [added](https://github.com/rails/rails/pull/45701) the `bin/rails routes --unused` command, I thought it makes sense to limit the unused routes

Reply:
Yeah, I think adding that might have been a mistake, if it drives people to overscope in their routes.rb. Meant to find routes for resources that don't exist at all any more.